build.sbt

```diff
@@ -36,6 +36,8 @@ addSbtPlugin("org.scalastyle" %% "scalastyle-sbt-plugin" % "0.8.0" excludeAll(
   ExclusionRule(organization = "com.danieltrinh")
 ))
 
+// If you change the scalariform version, you may also need to generate a new
+// scalariform.jar in src/main/resources/autoformat/
```

Comment:
Hmmm, I think the scalariform CLI is a different jar than what shows up in the ivy cache, since the scalariform jar is 1.9 M and the CLI (fat assembled jar) is ~ 8 M.